### DISCUSSION TITLE
Windows平台的类歌词字幕展示识别结果

### DISCUSSION THREAD

**Discussion by am009** [k2-fsa/sherpa-onnx#191] [Show and tell]
非常感谢这个项目，语音识别非常有用。

我们基于sherpa-onnx实现了一个Windows平台的实时抓取系统声音识别，并像听歌软件的歌词一样展示出来的小项目，更好地将语音识别应用起来。希望能帮到其他需要的人。

项目地址：https://github.com/jxlpzqc/TMSpeech   视频演示：https://www.bilibili.com/video/BV1rX4y1p7Nx/

![hover](https://github.com/k2-fsa/sherpa-onnx/assets/23314210/31eedfd3-14b0-4da8-8e95-5ee5079bef6f)

**Comment by csukuangfj** [comment]
> 非常感谢这个项目，语音识别非常有用。
> 
> 
> 
> 我们基于sherpa-onnx实现了一个Windows平台的实时抓取系统声音识别，并像听歌软件的歌词一样展示出来的小项目，更好地将语音识别应用起来。希望能帮到其他需要的人。
> 
> 
> 
> 项目地址：https://github.com/jxlpzqc/TMSpeech
> 
> 
> 
> ![hover](https://github.com/k2-fsa/sherpa-onnx/assets/23314210/31eedfd3-14b0-4da8-8e95-5ee5079bef6f)
> 
> 

非常棒！

有B站的视频演示吗？

**Reply by am009** [reply]
没，模型也打包好了，应该点开就能用，不知道需不需要视频演示。不过录一个应该也很容易

**Reply by csukuangfj** [reply]
希望能录一个，也好向大家宣传。 即使不安装，也能看到效果。

**Reply by am009** [reply]
视频演示有了：https://www.bilibili.com/video/BV1rX4y1p7Nx/

**Reply by csukuangfj** [reply]
谢谢！ 我可以去群里宣传下这个视频和这个项目吗

**Reply by am009** [reply]
可以可以，感谢宣传。
